GettingStarted/Tools.md

```diff
@@ -38,6 +49,18 @@ You should also install Slack on your laptop and phone, turn on @mention
 notifications, and respond to messages promptly. Slack has apps available for
 all major platforms.
 
+#### Cool Channels
+
+* `#programming-languages`: General discussion of programming languages and
+  related topics
+* `#graphics`: General discussion of computer graphics and animation
+* `#memes`: Name is self explanatory
```

Comment:
This was kinda snarky... might want to  rewrite this entry a bit.

Reply:
8aac11c ?

Reply:
Haha sure, resolving.